Applications/Internet/Internet Explorer 7.0/Online/script.js

```diff
@@ -31,6 +32,8 @@ var installerImplementation = {
             ])
             .set("builtin", ["updspapi"])
             .do();
+        
+        wine.run(setupFile, [], null, false, true);
```

Comment:
why did you add this line?

Reply:
Because in #627 you asked if I could add this implementation.

Reply:
No, sorry then you must have gotten me wrong. The issue here is (like in the SteamScript) that `run()` does not return a `Wine` object. Therefore, you cannot simply call `.wait()` after it. Instead, you must do something like:
```Javascript
var wine = new Wine()
               .run("iexplore", ["-unregserver"], null, false, true);
wine.overrideDLL()
...
```